src/nft/hooks/useWindowDimensions.ts

```diff
@@ -0,0 +1,28 @@
+import { useEffect, useState } from 'react'
```

Comment:
I think this is a duplicate of useWindowSize, in which case it is no longer needed.